modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/actions.es.js

```diff
@@ -102,19 +103,43 @@ export const dropLayoutBuilderField = ({
 };
 
 export const dropFieldSet = ({
+	availableLanguageIds = [],
 	dataLayoutBuilder,
 	fieldName,
-	fieldSet,
+	fieldSet: _fieldSet,
 	indexes,
 	parentFieldName,
 	useFieldName,
 	...otherProps
 }) => {
+	let fieldSet = _fieldSet;
```

Comment:
2. Could we create a const variable called `_fieldSet`, receiving the `fieldSet` props and use it?